src/SFA.DAS.FAA.Api/ApiResponses/GetApprenticeshipVacancyDetailResponse.cs

```diff
@@ -103,6 +102,8 @@ private static string GetDuration(ApprenticeshipVacancyItem source)
             }
 
             var duration = source.Duration == 0 ? source.Wage.Duration : source.Duration;
+            if (source.Wage is not {WageUnit: not null}) return source.ExpectedDuration;
```

Comment:
Double negative :)

Reply:
Re-written the condition & split the condition